server/app.js

```diff
@@ -258,9 +258,7 @@ const createApp = ({
   app.use(
     '/profile',
     createProfileRouter({
-      hubContentService,
       offenderService,
-      prisonerInformationService,
     }),
```

Comment:
are these no longer used by the profile route?

Reply:
The `createProfileRouter` definition, only pulls out the offenderService, from what I can see.
https://github.com/ministryofjustice/prisoner-content-hub-frontend/blob/3516af024d1f53b503819d32ba79daf702c02cf9/server/routes/profile.js#L3